code/mob/living/silicon/ai.dm

```diff
@@ -204,7 +204,7 @@ var/global/list/ai_emotions = list("Happy" = "ai_happy", \
 	APPLY_ATOM_PROPERTY(src, PROP_MOB_EXAMINE_ALL_NAMES, src)
 
 	ai_station_map = new /obj/minimap/ai
-	AddComponent(/datum/component/minimap_marker, MAP_AI, "ai")
+	AddComponent(/datum/component/minimap_marker, MAP_AI | MAP_SYNDICATE, "ai")
```

Comment:
ai sees all 👁️?

Reply:
Adding the `MAP_SYNDICATE` flag just means that the location of the AI core will be marked on minimaps with the same flag.